src/dxtbx/format/FormatNXmxEigerFilewriter.py

```diff
@@ -63,7 +84,10 @@ def get_raw_data(self, index):
 
 
 def get_raw_data(
-    nxdata: nxmx.NXdata, nxdetector: nxmx.NXdetector, index: int
+    nxdata: nxmx.NXdata,
+    nxdetector: nxmx.NXdetector,
+    index: int,
+    bit_depth: Optional[int] = None,
```

Comment:
```suggestion
    bit_depth: int | None = None,
```
We have `__future__.annotations` turned on here